### DISCUSSION TITLE
getting an ERR_CERT_COMMON_NAME_INVALID on custom server

### DISCUSSION THREAD

**Discussion by chrisr17** [vercel/community#5155] [Help] [answered]
### Summary

getting a cert error on my custom server

### Example

bigblockmodular.ca

### Steps to Reproduce

navigate to bigblockmodular.ca

**Comment by amyegan** [comment]
Hi @chrisr17. I don't get any errors when I visit the site. Can you share a screenshot of the error you see?

**Comment by chrisr17** [comment, marked as answer]
hey amy, it took care of itself after some time... so it appears i was 
being impatient. sorry for the grief. chris

On 2023-12-11 12:22, Amy wrote:

> Hi @chrisr17 [1]. I don't get any errors when I visit the site. Can you 
> share a screenshot of the error you see?
> 
> --
> Reply to this email directly, view it on GitHub [2], or unsubscribe 
> [3].
> You are receiving this because you were mentioned.Message ID: 
> ***@***.***>


Links:
------
[1] https://github.com/chrisr17
[2] 
https://github.com/orgs/vercel/discussions/5155#discussioncomment-7823082
[3]
